onnxruntime/core/providers/opencl/README.md

```diff
@@ -82,6 +82,33 @@ python tools/ci_build/github/android/build_aar_package.py --config Release openc
 
 The resulting aar package is a ready dependency for gradle project.
 
+## Run model
+
+### .onnx file format
+
+You must not enable `--minimal_build` or related options during building to
+support onnx file format.
+
+### .ort file format
+
+1. Build a full package of onnxruntime with `--build_wheel`, this will install
+   the wheel package files to `<build_dir>/<OS>/<config>/build/lib/`, say the
+   path is `~/onnxruntime/build/Linux/Debug/build/lib/`
```

Comment:
Not sure this is the case on all platforms. On Windows it creates the directory with the python module in <build_dir>/<OS>/<config>/<config>/onnxruntime

Reply:
Done. Sub-sectioned for  Windows and *nix